client-ts/Microsoft.AspNetCore.SignalR.Client.TS/HubConnection.ts

```diff
@@ -196,3 +210,38 @@ export class HubConnection {
         };
     }
 }
+
+class Base64EncodedHubProtocol implements IHubProtocol {
+    private wrappedProtocol: IHubProtocol;
+
+    constructor(protocol: IHubProtocol) {
+        this.wrappedProtocol = protocol;
+        this.name = this.wrappedProtocol.name;
+        this.type = ProtocolType.Text;
+    }
+
+    readonly name: string;
+    readonly type: ProtocolType;
+
+    parseMessages(input: any): HubMessage[] {
+        // atob/btoa are browsers APIs but they can be polyfilled. If this becomes problematic we can use
```

Comment:
Are we going to do anything about this though? I did try base64-js package and reverted the change since it added 2.2 KB to the library (minified) which is wasteful on browsers since the functionality is built-in. If you hit this on node you would need to polyfill. Unless the bug is supposed to be a doc bug which will be turned into a an article about dependencies a user needs to polyfill when running on node.